src/components/FilmCard.vue

```diff
@@ -11,8 +11,8 @@
           img-alt="Image"
           img-top
           tag="article"
-          class="text-muted card-text form-control card-img-top b-card "
-          @click="$emit('getCardId', cardId)"
+          class="text-muted card-text form-control card-img-top b-card"
+          @click="$emit('getCardId', cardId), $store.dispatch('getActors', cardId)"
```

Comment:
Don't dispatch actions directly.

Use the `mapActions()` with a spread operator inside `methods:`. Then call the action by it's action name.